apps/api/src/app/environments/environments.controller.ts

```diff
@@ -68,4 +81,15 @@ export class EnvironmentsController {
 
     return await this.getApiKeysUsecase.execute(command);
   }
+
+  @Put('/encrypt')
```

Comment:
If we are still going to the specific endpoint to update configurations for that. Maybe worth calling this:
```suggestion
  @Put('/security')
```

Reply:
I will create  /widget/settings for future usage.